packages/code-du-travail-frontend/src/layout/Footer.js

```diff
@@ -129,10 +131,13 @@ const ContainerWrapper = styled.div`
   text-align: left;
 `;
 
-const Links = styled.div`
+const LinkSection = styled(Section)`
+  box-shadow: 0 -10px 10px -10px ${colors.lightGrey};
```

Comment:
can we add it to theme to have consistent shadow ?

Reply:
It's a one time use. I doubt we will need elsewhere